hist/hist/src/TFormula_v5.cxx

```diff
@@ -2685,10 +2686,10 @@ Double_t TFormula::EvalParOld(const Double_t *x, const Double_t *uparams)
 {
    Int_t i,j;
    // coverity[uninit] the tab value of tab is guaranteed to be set properly by the control flow.
-   Double_t tab[kMAXFOUND];
-   const char *stringStack[gMAXSTRINGFOUND];
+   Double_t tab[kMAXFOUND] = {0};
```

Comment:
These initializations cost time. @lmoneta - are they actually needed or is Coverity incorrect here? The comment seems to claim Coverity is wrong.